index.bs

```diff
@@ -349,6 +349,220 @@ One example of a feature that should be limited to secure contexts
 is geolocation, since the authentication and confidentiality provided by
 secure contexts reduce the risks to user privacy.
 
+<h2 id="css">Cascading Style Sheets (CSS)</h2>
+
+<h3 id="css-versus-markup">Presentation should be in CSS, content should be in HTML</h3>
+
+The separation between CSS and HTML was originally designed around the idea
```

Comment:
Perhaps do 'CSS (style) and HTML (markup)' here, since you use the terms interchangeably after this? Or be consistent about using CSS and HTML instead of style and markup.

Reply:
fixed